netbench/cdk/src/main/java/com/aws/StateMachineStackProps.java

```diff
@@ -14,9 +17,21 @@ public static Builder builder() {
 
     EcsRunTask getClientTask();
 
+    Bucket getBucket();
```

Comment:
similar comment to previous. Lombok can do everything in this file and reduce it to a couple of lines